src/schema/v2/Feature/FeatureImageType.ts

```diff
@@ -0,0 +1,75 @@
+import {
+  GraphQLString,
+  GraphQLObjectType,
+  GraphQLInt,
+  GraphQLNonNull,
+} from "graphql"
+import { Gravity } from "@artsy/quartz"
+import { scale } from "proportional-scale"
+import { ResolverContext } from "types/graphql"
+import proxy from "schema/v2/image/proxies"
+import { ScaledImage, TScaledImage } from "../image"
+
+export const FeatureImageType = new GraphQLObjectType<
+  Gravity.Feature,
+  ResolverContext
+>({
+  name: "FeatureImage",
+  fields: {
+    url: {
+      type: GraphQLString,
+      args: {
+        version: { type: GraphQLString, defaultValue: "wide" },
+      },
+      resolve: ({ image_urls }, args) => image_urls[args.version],
+    },
+    resized: {
+      type: ScaledImage,
+      args: {
+        width: { type: GraphQLInt },
+        height: { type: GraphQLInt },
+        quality: { type: GraphQLInt, defaultValue: 80 },
+        version: {
+          type: new GraphQLNonNull(GraphQLString),
```

Comment:
The thing of it is that they are possibly undefined for literally every field, so: we can't define them via enum since we only know them at runtime and on a per-request basis. We like sort of know what MIGHT be there MOST of the time. Though this varies from model to model. 🤕